homeassistant/components/tesla.py

```diff
@@ -0,0 +1,108 @@
+"""
+Support for Tesla cars.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/vera/
+"""
+from collections import defaultdict
+
+import voluptuous as vol
+
+from homeassistant.util import slugify
+from homeassistant.helpers import discovery
+from homeassistant.helpers import config_validation as cv
+from homeassistant.const import (
+    ATTR_ARMED, ATTR_BATTERY_LEVEL)
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['teslajsonpy==0.0.9']
+
+DOMAIN = 'tesla'
+
+TESLA_CONTROLLER = None
+
+CONF_EMAIL = 'email'
+CONF_PASSWORD = 'password'
+CONF_UPDATE_INTERVAL = 'update_interval'
```

Comment:
Please use `scan_interval` instead of `update_interval` (already declared in `homeassistant.const` as `CONF_SCAN_INTERVAL`).